docs/scos/dev/glue-api-guides/202311.0/migrating-old-resource-to-new-infrastructure.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Migration of Legacy Glue API resource to Glue Storefront API infrastructure
```

Comment:
Context is missing

Why should I migrate? 
What are the benefits? 

we always talk about that Legacy Glue can be fallback but how? and why is this not part of this document?